### PR TITLE
Close #62 and #63 Updated vignette with 24.1 CL rationale and code format ADAPET, ADTPET…

### LINKED ISSUE #62: Documentation:  Review vignete adpet.Rmd for consistent use of ADAPET, ADTPET, PET etc
### Please select a category the issue is focused on?

Function Documentation

### Let us know where something needs a refresh or put your idea here!

Per rossfarrugia:

in https://pharmaverse.github.io/admiralneuro/articles/adpet.html use consistent approach to how you show ADaM names, i.e. "consulting the `ADTPET` and `ADAPET` template" instead of "consulting the ADTPET and ADAPET template". Also use consistent casing for PET, e.g. see "basis of the Pet Scan ADaM datasets"

### PR DESCRIPTION
**Pull request by jwang-lilly**
… and change Pet to PET

Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral family codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral<ext> (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: vignettes/adpet.Rmd [hunk 1]
```diff
@@ -21,21 +21,11 @@ library(admiraldev)
...
-vignette](https://pharmaverse.github.io/admiral/articles/bds_finding.html).
-The programming workflow around creating the general set-up of a BDS ADaM dataset using
-`{admiral}` functions is the same. In this vignette, we focus on the neuro-specific steps only to avoid repeating
-information and maintaining the same content in two places. As such, the code in this vignette is not
-completely executable; we recommend consulting the ADTPET and ADAPET template scripts
-to view the full workflow.
-
-**Note**: *All examples assume CDISC SDTM and/or ADaM format as input
-unless otherwise specified.*
+We recommend splitting out amyloid and tau into two distinct ADaMs datasets. The vignette presents a combined approach for the initial setup, parameter mapping and finals steps. It then splits into separate sections for each data type, as they require different processes: starting with `ADTPET` (Tau PET Scan Analysis Dataset), followed by `ADAPET` (Amyloid PET Scan Analysis Dataset).
```
Comment (khatril) [inline vignettes/adpet.Rmd:24]:
```suggestion
We recommend splitting out amyloid and tau into two distinct ADaM datasets. The vignette presents a combined approach for the initial setup, parameter mapping and finals steps. It then splits into separate sections for each data type, as they require different processes: starting with `ADTPET` (Tau PET Scan Analysis Dataset), followed by `ADAPET` (Amyloid PET Scan Analysis Dataset).
```

File: vignettes/adpet.Rmd [hunk 2]
```diff
@@ -333,13 +297,13 @@ adapet_param <- distinct(adapet %>% filter(NVTESTCD == "SUVR" | !is.na(AVALU)),
...
 
-### Derive Criterion Flag Variables for Amyloid Categories{#criterionfl}
+### Derive Criterion Flag Variables for Amyloid Categories {#criterionfl}
 
 Having converted the `SUVR` to Centiloid, we can now derive the criterion flags for the amyloid categories.
 
-To do this, we will use the `admiral::derive_vars_crit_flag()` function.
-Since we want to derive these flags specifically for the Centiloid parameters,
-we will apply the derivation only to those records using `admiral::restrict_derivation()`.
+We will use Centiloid cutoff of **24.1** to define amyloid negative and positive categories. The 24.1 Centiloid cutoff comes from an autopsy-validated study in which amyloid PET SUVR values were compared with post-mortem brain amyloid plaque counts. The SUVR threshold corresponding to moderate-to-frequent neuritic plaques under the Consortium to Establish a Registry for Alzheimer’s Disease (CERAD) criteria was mapped to the Centiloid scale, yielding 24.1 CL as the pathology-based definition of amyloid positivity. Centiloid \>= 24.1 is considered amyloid positive, while Centiloid \< 24.1 is considered amyloid negative.
```
Comment (khatril) [inline vignettes/adpet.Rmd:304]: Should we include a reference to this study?

**Review by khatril — COMMENTED** [review]
Thanks @jwang-lilly Looks great, some very minor comments

**Comment by manciniedoardo** [comment]
@jwang-lilly can you please double-check the link as it's giving a 404

**Comment by khatril** [comment]
> @manciniedoardo, I tried the three links to Navitsky paper, all work from my end. Not sure why the link check failed. https://doi.org/10.1016/j.jalz.2018.06.1353 https://alz-journals.onlinelibrary.wiley.com/doi/10.1016/j.jalz.2018.06.1353 https://www.sciencedirect.com/science/article/abs/pii/S1552526018315255

@jwang-lilly Maybe we can put a citation to this paper instead of the direct link. It's possible that the link to the published paper may require a subscription, but I haven't encountered this before either.
